lib/client/components/Location.jsx

```diff
@@ -1,5 +1,6 @@
 'use strict';
 import React from 'react';
+var Geolocation = require('../util/Geolocation');
```

Comment:
Jest claims to mock all the DOM APIs, but I could not find the `navigator` object at runtime.  I may have missed something and would be happy if there was a way to get rid of this.